examples/website/src/docs/db/sqlite.nim

```diff
@@ -14,14 +14,14 @@ component SQLite:
 
       case currentLanguage.val
       of "Nim":
-        tP: {translate"This article discusses the interaction of Happy and SQLite using the NORM library."}
+        tP: {translate"This article discusses the interaction of Happy and SQLite using Norm library."}
```

Comment:
Here you're use `Norm`, but in `translations.nim` here uses `the Norm`. It shouldn't to be different (`the Norm` or `Norm` only). It need to translations works